storybook/stories/Loader.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Loader, Alert, Container, Touchable } from "@components";
+
+const LoaderStories = {
+  title: "Loaders",
+  component: Loader,
+};
+
+export default LoaderStories;
+
+export const Loaders = () => {
+  return (
+    <Container justifyContent="space-around" flex={1}>
+      <Alert />
+      <Container flexDirection="row">
+        <Loader />
+        <Loader size="m" colorVariant="primary" />
+        <Loader size="l" colorVariant="primary" />
+      </Container>
+      <Container
+        flexDirection="row"
+        width="100%"
+        justifyContent="space-between"
+        alignItems="center"
```

Comment:
@kiran-kumar011 Replace all the components which are using `<ActivityIndicator/>` from rn with our new loader.

Reply:
Hi @sangameshsomawar ,
Can I create a separate task for this.

I replaced the ActivityIndicator with Loader but when the width is not defined the Loader wrapper will take the complete width and center the loader.

Reply:
@kiran-kumar011 It should not take complete width. It should behave same as activity indicator from react native. We can create separate issue but still it should not take full width.

Reply:
@sangameshsomawar ,
I have started working on the same branch and fixed all the issue related to alignment, now the Loader component behaves exactly similar to ActivityIndicator.

Verified the issues locally different components and fixed the issue.